tests/tools/ingest_historical_csv_test.py

```diff
@@ -0,0 +1,12 @@
+"""To test historical data insertion script"""
```

Comment:
We can remove this file if not have any code...otherwise we will forgot to write testcase as its checked-in